tests/api/mocks.py

```diff
@@ -0,0 +1,117 @@
+from v3io.dataplane.kv import Model
```

Comment:
I feel like this will get longer and longer and we better already separate to different files
please make `mocks` package and under it file for each mock class

Reply:
Will be iterated on in next PR, removing for now